### PR TITLE
Adding svdb v1.0.7

### PR DESCRIPTION
**Pull request by holtgrewe**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by holtgrewe** [comment]
Hm, @bioconda/python does anyone see what's wrong here? I'm clueless. Building appears to fail for `scikit-learn` but it's in my `meta.yaml`...

**Comment by mbargull** [comment]
What happened here is that `conda` installed `scikit-learn 0.19.1`, but `setuptools` wants `scikit-learn ==0.15.2`. You'll have to use
```bash
python setup.py install --single-version-externally-managed --record record.txt
```
to prevent `setuptools` from trying to download dependencies by itself. And it's best to also use
```yaml
    - scikit-learn ==0.15.2
```
in `meta.yaml` as well, if that version constraint is really needed. If it's not needed, you should patch `setup.py` to remove that constraint.

File: recipes/svdb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+{% set md5sum = "2764c03866f3aa827bec3fcc50fa1742" %}
+
+package:
+  name: svdb
+  version: "{{ version }}"
+
+source:
+  fn: svdb-{{ version }}.tar.gz
+  url: https://github.com/J35P312/SVDB/archive/SVDB-{{ version }}.tar.gz
+  md5: "{{ md5sum }}"
```
Comment (mbargull) [inline recipes/svdb/meta.yaml]: It's [usually preferred to use `sha256`](http://bioconda.github.io/guidelines.html#hashes).

File: recipes/svdb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+build:
+  skip: True  # [not py27]
+  number: 0
```
Comment (mbargull) [inline recipes/svdb/meta.yaml]: Can you add
```yaml
  entry_points:
    - svdb = svdb.__main__:main
```
to the build section.

File: recipes/svdb/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  # missing dependency of scipy
+  - libgcc
+
+test:
+  commands:
+  - svdb --help
+
+about:
+  home: https://github.com/J35P312/SVDB
+  license: MIT
```
Comment (mbargull) [inline recipes/svdb/meta.yaml]: I couldn't find the license declaration in `setup.py` or `README.md` -- are you sure it's MIT licensed?

File: recipes/svdb/meta.yaml (hunk 1 above)
Comment (mbargull) [inline recipes/svdb/meta.yaml]: It's [usually preferred to use `sha256`](http://bioconda.github.io/guidelines.html#hashes).

File: recipes/svdb/build.sh [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```
Comment (mbargull) [inline recipes/svdb/build.sh]: Use
```bash
python setup.py install --single-version-externally-managed --record record.txt
```

File: recipes/svdb/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  skip: True  # [not py27]
+  number: 0
+
+requirements:
+  build:
+  - cython
+  - python
+  - numpy
+  - scipy
+  - scikit-learn
```
Comment (mbargull) [inline recipes/svdb/meta.yaml]: Use
```yaml
    - scikit-learn ==0.15.2
```
or patch `setup.py`.

File: recipes/svdb/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  # missing dependency of scipy
+  - libgcc
+
+  run:
+  - python
+  - numpy
+  - scipy
+  - scikit-learn
```
Comment (mbargull) [inline recipes/svdb/meta.yaml]: Use
```yaml
    - scikit-learn ==0.15.2
```
or patch `setup.py`.

File: recipes/svdb/meta.yaml (hunk 3 above)
Comment (mbargull) [reply recipes/svdb/meta.yaml]: Ah, found the [issue you created](https://github.com/J35P312/SVDB/issues/12), thanks!

File: recipes/svdb/meta.yaml (hunk 2 above)
Comment (mbargull) [reply recipes/svdb/meta.yaml]: Argh, I wasn't notified of your update so I didn't review them/approve them (maybe GitHub doesn't notify if no commit was added and force pushed -- strange thing...). That `entry_points:` should've been part of the build section.

Comment (holtgrewe) [reply recipes/svdb/meta.yaml]: Gosh, ok, let's continue this on #7565?

**Pull request merged**